pkgs/development/tools/electron/default.nix

```diff
@@ -1,171 +1,96 @@
-{ lib, stdenv
-, libXScrnSaver
-, makeWrapper
-, fetchurl
-, wrapGAppsHook
-, glib
-, gtk3
-, unzip
-, atomEnv
-, libuuid
-, at-spi2-atk
-, at-spi2-core
-, libdrm
-, mesa
-, libxkbcommon
-, libappindicator-gtk3
-, libxshmfence
-, libglvnd
-, wayland
-}@args:
+{ lib, chromium, fetchFromGitHub, jq, git, nodejs, fetchYarnDeps, fixup_yarn_lock, yarn, substituteAll, libnotify, unzip }:
 
 let
-  mkElectron = import ./generic.nix args;
+  info = lib.importJSON ./info.json;
 in
-rec {
 
-  electron = electron_23;
+(chromium.override { channel = "electron_23"; }).mkDerivation (base: rec {
+  name = "electron";
```

Comment:
```suggestion
  pname = "electron";
```

Reply:
this is specific to chromium.mkDerivation

Reply:
changed to "packageName"